app/controllers/appropriate_bodies/process_batch/claims_controller.rb

```diff
@@ -18,25 +18,34 @@ def create
 
           record_bulk_upload_started_event
           process_batch_claim
-          record_bulk_upload_completed_event
 
-          redirect_to ab_batch_claim_path(@pending_induction_submission_batch), alert: 'File processing'
+          redirect_to ab_batch_claim_path(@pending_induction_submission_batch)
         else
           csv_data.errors.each do |error|
             @pending_induction_submission_batch.errors.add(error.attribute, error.message)
           end
+
           render :new, status: :unprocessable_entity
         end
       rescue ActionController::ParameterMissing
-        @pending_induction_submission_batch.errors.add(:csv_file, "Attach a CSV file")
+        @pending_induction_submission_batch.errors.add(:csv_file, 'Select a file')
         render :new, status: :unprocessable_entity
-      rescue StandardError => e
-        @pending_induction_submission_batch.errors.add(:base, e.message)
+      rescue StandardError
+        @pending_induction_submission_batch.errors.add(:base, 'Something went wrong')
```

Comment:
I wonder if this is a good idea. The sooner we find out (via Sentry) something bad happened the sooner we find out and can react.

Reply:
I think it was useful for a time but now it is stable I agree. I'll remove and see how a retest goes.